sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CTrustedIssuerRepository.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.autoconfigure.b2c;
+
+import com.azure.spring.aad.webapi.AADTrustedIssuerRepository;
+import org.springframework.util.Assert;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class AADB2CTrustedIssuerRepository extends AADTrustedIssuerRepository {
+
+    private String baseUri;
```

Comment:
can we replace the baseUri with `resolvedBaseUri` here, since I see baseUri  is not used directly